src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -237,7 +237,10 @@ def on_time_step(self, event: Event):
                 pop["household_id"]
                 == data_values.NONINSTITUTIONAL_GROUP_QUARTER_IDS["Military"]
             )
-            & (pop["age"] >= data_values.WORKING_AGE)
+            & (
+                pop["age"]
+                >= data_values.WORKING_AGE - event.step_size.days / utilities.DAYS_PER_YEAR
+            )
```

Comment:
This catches sims turning working age that live in military gq.  Same thing as turning_working_age but we would not update them here to have military employer.

Reply:
I don't understand why this is necessary. Simulants age on time-step prepare, so they should already be of working age.

Reply:
Simulants age on time step clean up.

Reply:
Gotcha. I'd rather leave the simulation code simple (i.e. leave this line as is) and make the test only test people who are more than one time-step older than 18.

Reply:
Ok to confirm - update the test.  This will mean we will have unemployed simulants in the military for one time step because in time step cleanup they will turn 18 then the next time step they then will have their employer_id updated to military because they will not be included in the working_age_idx.

Reply:
This will mean the test will add a time step to the appropriate age.